packages/atomic/src/components/search/facets/atomic-popover/atomic-popover.tsx

```diff
@@ -0,0 +1,196 @@
+  @InitializeBindings()
+  public bindings!: Bindings;
+  public searchStatus!: SearchStatus;
+  @BindStateToController('searchStatus')
+  @State()
+  public searchStatusState!: SearchStatusState;
+  @State()
+  public error!: Error;
+
+  @State() public isMenuVisible = false;
+  @State() public facetId?: string;
+  @State() public facetLabel?: string = 'no-label';
+  @Element() host!: HTMLElement;
+  private facetElement?: HTMLElement;
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+
+    if (this.host.children.length === 0) {
+      this.error = {
+        name: 'No child in popover',
+        message: 'One child is required inside a set of popover tags',
+      };
+    } else if (this.host.children.length > 1) {
+      this.error = {
+        name: 'Too many children in popover',
+        message: 'Cannot have more than one child inside a set of popover tags',
+      };
+    }
+  }
+
+  private get popoverId() {
+    return `${this.facetId}-popover`;
+  }
+
+  popoverOpened() {
+    if (!this.isMenuVisible) {
+      const popoverOpened = new CustomEvent<ClearPopoversEventPayload>(
+        'clearPopovers',
+        {
+          detail: {ignorePopoverFacetId: this.facetId},
+        }
+      );
+      document.dispatchEvent(popoverOpened);
+    }
+
+    this.isMenuVisible = !this.isMenuVisible;
+  }
+
+  @Listen('clearPopovers', {target: 'document'})
+  popOpened(event: CustomEvent<ClearPopoversEventPayload>) {
+    if (event.detail.ignorePopoverFacetId !== this.facetId) {
+      this.isMenuVisible = false;
+    }
+  }
+
+  renderValueButton() {
+    return (
+      <Button
+        style="square-neutral"
+        onClick={() => this.popoverOpened()}
+        part="label-button"
+        ariaExpanded={`${this.isMenuVisible}`}
+        ariaLabel={`Pop-up filter on ${this.facetLabel} facet`}
+        ariaControls={this.popoverId}
+        class={`rounded flex box-border h-full items-center mr-1.5 p-2.5 group ${
+          this.isMenuVisible
+            ? 'selected border-primary ring ring-ring-primary-light text-primary'
+            : 'hover:border-primary-light focus-visible:border-primary-light '
+        }`}
+      >
+        <span
+          title={this.facetLabel}
+          part="label"
+          class={`truncate ${
+            this.isMenuVisible
+              ? ''
+              : 'group-hover:text-primary-light group-focus:text-primary'
+          }`}
+        >
+          {this.facetLabel}
+        </span>
+        <atomic-icon
+          part="arrow-icon"
+          aria-hidden="true"
+          class={`w-2 ml-2 ${
+            this.isMenuVisible
+              ? 'rotate-180'
+              : 'group-hover:text-primary-light group-focus:text-primary'
+          } `}
+          icon={ArrowBottomIcon}
+        ></atomic-icon>
+      </Button>
+    );
+  }
+
+  @Listen('facetInitialized')
+  injectPopoverClass(event: CustomEvent<InitPopoverEventPayload>) {
```

Comment:
Perhaps a better name would be linkFacet or something similar, because there is more to it than injecting the class